openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/CacheConfigAnnotation.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.ballerina.openapi.converter.service;
+
+import java.util.ArrayList;
+import java.util.List;
+/**
+ * This class uses to store all the cache configuration details.
+ *
+ * @since 2.0.0
+ */
+public class CacheConfigAnnotation {
+    private boolean mustRevalidate = false;
```

Comment:
In the default `@http:CacheConfig`, we have enabled the `must-revalidate` directive as well. So can we change this to `true`

Reply:
Sure let's change with it :). There is a concern about this if some user doesn't set must-validate in their cache-config ex:
```ballerina 
resource function get cachingBackEnd(http:Request req) returns @http:CacheConfig{maxAge : 5} string {
       return "Hello, World!!";}
```
what would be a default string value we get ? is it similar to this `public, max-age=5` or `public, max-age=5, must-revalidate` ?

Reply:
It will be `public, max-age=5, must-revalidate`

Reply:
Thank you , will change with this :)

Reply:
Fix with https://github.com/ballerina-platform/ballerina-openapi/pull/564/commits/880d1122b2fc897bb656d1f44c31bdf3a7906695